docs/database/database-spans.md

```diff
@@ -56,25 +56,25 @@ Database spans MUST follow the overall [guidelines for span names](https://githu
 
 <!-- markdown-link-check-disable -->
 <!-- HTML anchors are not supported https://github.com/tcort/markdown-link-check/issues/225-->
-The **span name** SHOULD be `{db.operation.name} {target}` if there is a
-(low-cardinality) `{db.operation.name}` available (see below for the exact definition of the [`{target}`](#target-placeholder) placeholder).
+The **span name** SHOULD be `{db.operation.name} {entity}` if there is a
+(low-cardinality) `{db.operation.name}` available (see below for the exact definition of the [`{entity}`](#entity-placeholder) placeholder).
 
 If there is no (low-cardinality) `db.operation.name` available, database span names
-SHOULD be [`{target}`](#target-placeholder).
+SHOULD be [`{entity}`](#entity-placeholder).
 <!-- markdown-link-check-enable -->
 
-If neither `{db.operation.name}` nor `{target}` are available, span name SHOULD be `{db.system}`.
+If neither `{db.operation.name}` nor `{entity}` are available, span name SHOULD be `{db.system}`.
 
 Semantic conventions for individual database systems MAY specify different span name format.
 
-The <span id="target-placeholder">`{target}`</span> SHOULD describe the entity that the operation is performed against
+The <span id="entity-placeholder">`{entity}`</span> SHOULD describe the entity that the operation is performed against
 and SHOULD adhere to one of the following values, provided they are accessible:
 
-- `db.collection.name` SHOULD be used for data manipulation operations or operations on database collections.
+- `db.operation.target` SHOULD be used for data manipulation operations or operations on database collections.
 - `db.namespace` SHOULD be used for operations on a specific database namespace.
```

Comment:
I think it's confusing:

E.g. I have `CREATE DATABASE` query or `cosmosDatabase.listContainers()` operation, the `db.operation.target` semantics would imply that database namespace is a target.

I believe we still want distinct attributes for db namespace and table/container/collection/index(es).